### DISCUSSION TITLE
Disable unneeded imageformat support on Windows

### DISCUSSION THREAD

**Discussion by dmitri-fm** [ImageMagick/ImageMagick#5393] [Help]
I'm building ImageMagick as static library, and the size of some libraries (CORE_exr, CORE_jpeg_xl) is too big (up to 50 MB).
It also makes my final exe too big too.
Is there a way to get more control of configuration if I'm building ImageMagick for Windows (https://github.com/ImageMagick/ImageMagick-Windows)?
I would like to disable exr, tiff, maybe other formats.

**Comment by dlemstra** [comment]
You can remove the folders of the formats that you don't want from the `VisualMagick` folder.

**Comment by dmitri-fm** [comment]
I tried, but now I have missing references in other modules (CORE_RL_rsvg).
![img](https://i.imgur.com/dnTNqxZ.png)
Do you have a list of folders which can be deleted without any problems?

**Reply by dlemstra** [reply]
You will need to figure out which project depends on which project yourself and then decide what you want to remove from the build.

**Comment by dmitri-fm** [comment]
Can I remove that rsvg also? I guess it's related to rendering (which I don't need)?

**Reply by dlemstra** [reply]
It is used to read svg files and I don't know if you need that feature.
